pip_audit/_cli.py

```diff
@@ -280,11 +287,34 @@ def audit() -> None:
                 pkg_count += 1
                 vuln_count += len(vulns)
 
+    # If the `--fix` flag has been applied, find a set of suitable fix versions and upgrade the
+    # dependencies at the source
+    fixes = list()
+    fixed_pkg_count = 0
+    fixed_vuln_count = 0
+    if args.fix:
+        for fix_version in resolve_fix_versions(service, result):
+            if not fix_version.is_skipped():
+                fix_version = cast(ResolvedFixVersion, fix_version)
+                try:
+                    source.fix(fix_version)
+                    fixed_pkg_count += 1
+                    fixed_vuln_count += len(result[fix_version.dep])
+                except DependencySourceError as dse:
+                    fix_version = SkippedFixVersion(fix_version.dep, str(dse))
+            fixes.append(fix_version)
```

Comment:
These will eventually get passed into the `VulnerabilityFormat` interface.

Reply:
SGTM!

xref #215.